coreldraw_cdr.ksy

```diff
@@ -2655,3 +3120,70 @@ types:
             - cdrSVGPalette
             - SVGColor # file name (SVGColor.xml)
           doc: SVG Colors
+  text_style_flags:
+    seq:
+      - id: emphasis_raw
+        type: u2
+      - id: unknown
+        type: b2
+      - id: underline
+        type: b3
+        enum: decoration
+      - id: overline
+        type: b3
+        enum: decoration
+      - id: strike_through_line
+        type: b3
+        enum: decoration
+      - id: script
+        type: b2
+        enum: script_type
+      - type: b3
```

Comment:
This unknown field apparently corresponds to the [`cdrFontCase` enum](https://community.coreldraw.com/sdk/api/draw/17/e/cdrfontcase). Sample file here: [**font-caps-x7.zip**](https://github.com/kaitai-io/coreldraw_cdr.ksy/files/10396686/font-caps-x7.zip)

In theory, values of `cdrFontCase` would fit into just 2 bits, so it's unclear what's with that 1 remaining bit. We can probably leave it like this for now, though, and once we encounter files with strange values here, we can correct it.